external/maybe-sync-external.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview
+ */
+
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+
+// Synchronize remote data at most once every 12 hours.
+const syncThresholdMs = 12 * 60 * 60 * 1000;
```

Comment:
I wonder if it wouldn't be possible to use eleventy-cached-data plugin here instead of custom cashing.

Reply:
I think this is to determine weather to write data to the storage bucket.